python/caffe/_caffe.cpp

```diff
@@ -275,7 +275,7 @@ BOOST_PYTHON_MODULE(_caffe) {
   bp::class_<vector<bool> >("BoolVec")
     .def(bp::vector_indexing_suite<vector<bool> >());
 
-  import_array();
+  import_array1();
```

Comment:
Can you explain/reference this? Everything I've found defines `import_array1` as a unary function (hence the 1), and I can't find any reason there'd be a difference between 2 and 3, so I'm confused as to why this works at all.